### PR TITLE
chore: adding code owner for confirmation domain's code

### PR DESCRIPTION
**Pull request by jpuri**
## **Description**

Adding code owner for confirmation domain's code

## **Related issues**

Fixes: MetaMask/MetaMask-planning#2261

## **Manual testing steps**
NA

## **Screenshots/Recordings**
NA

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by pedronfigueiredo — DISMISSED** [review]

**Review by cryptotavares — DISMISSED** [review]

File: .github/CODEOWNERS
Comment (matthewwalsh0) [inline .github/CODEOWNERS:56]: Can we include the @MetaMask/confirmations-system-team here also?

Wwe maintain the generic `ConfirmationPage` as part of the approval framework, plus we're currently working on alerts and simulations that will have entries within here.

Comment (brad-decker) [reply .github/CODEOWNERS:56]: I can approve once this change is made

Comment (jpuri) [reply .github/CODEOWNERS]: I made the update, but found this issue for both the teams: ux and system. We need to fix think before we merge:

**Review by brad-decker — APPROVED** [review]

File: .github/CODEOWNERS
Comment (brad-decker) [reply .github/CODEOWNERS]: Resolved, @jpuri - you all have write access via other methods but the team needs write access in order to be listed as codeowners, which you now have.

**Review by cryptotavares — APPROVED** [review]

File: .github/CODEOWNERS
Comment (jpuri) [reply .github/CODEOWNERS]: Thanks @brad-decker

**Review by pedronfigueiredo — APPROVED** [review]

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**
